main.go

```diff
@@ -20,6 +20,7 @@ package main
 
 import (
 	"flag"
+	"os"
```

Comment:
This shouldn’t be necessary (but Go linting should catch it).

Reply:
Was having some problems with IDE, it kept freezing on trying to save so had couple commits without changes I wanted.